qemu/tests/cfg/passthrough_with_multipath_device.cfg

```diff
@@ -0,0 +1,24 @@
+- passthrough_with:
+    virt_test_type = qemu
+    type = passthrough_with_multipath_device
+    kill_vm_on_error = yes
+    login_timeout = 240
+    start_vm = yes
+    only Linux
+    variants:
+        - multipath_device:
+            pre_command = "yum install device-mapper-multipath -y;mpathconf --enable;systemctl restart multipathd;"
+            pre_command += "modprobe scsi_debug dev_size_mb=5000 num_tgts=1 vpd_use_hostno=0 add_host=2 delay=1 max_luns=2 no_lun_0=1;sleep 5"
+            post_cmd = "multipath -F;sleep 3;modprobe -r scsi_debug"
+            images += " stg0"
+            image_name_stg0 = "/dev/mapper/mpatha"
```

Comment:
We need to find the mpath instead of fixedsdssds4seeedseseewwww name, due to some hosts have exist other multipath already.